styles.css

```diff
@@ -0,0 +1,420 @@
+
+.in-magazine .top-title,
+.editor-picks .top-title,
+.featured-slideshows .top-title {
+  font-weight: bold;
+  color: #000;
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: 0.8rem;
+}
+
+.see-features {
+  background: #f4f5f6;
+  font-weight: bold;
+  color: #000;
+  padding: 0.5rem 0.8rem;
+}
+
+.in-magazine .top-title img,
+.editor-picks .top-title img,
+.featured-slideshows .top-title img {
+  max-width: 20px;
+  margin-right: 0.4rem;
+  position: relative;
+  top: -3px;
+}
+
+.cat-row {
+  border-bottom: 1px solid #e5e5e5;
+}
+
+.categories {
+  margin: 4rem 0 2rem 0;
+  border-top: 1px solid #e5e5e5;
+  padding: 2rem 0;
+}
+
+.cat-title {
+  color: #f72210;
+  font-weight: bold;
+}
+
+.category-box .third-col .post {
+  padding-top: 0.7rem;
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: 0.7rem;
+}
+
+.nw-img {
+  height: 100%;
+  object-fit: cover;
+}
+
+.newsletter-row {
+  border: 1px solid #e5e5e5;
+  max-width: 98%;
+  margin: auto;
+}
+
+.subscribe-sec .story-title {
+  font-weight: bold;
+  margin-top: 1.4rem;
+}
+
+.subscribe-sec {
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: 2rem;
+}
+
+.social li {
+  list-style: none;
+}
+
+.footer-navigation li {
+  list-style: none;
+  float: left;
+}
+
+.copyright-row {
+  max-width: 95%;
+  margin: 0 auto 1rem auto;
+  border-bottom: 1px solid #e5e5e5;
+}
+
+.footer-logo {
+  max-width: 220px;
+}
+
+.btn-red {
+  color: #fff;
+  border-radius: 0;
+  padding: 0.5rem 1.2rem;
+  font-weight: bold;
+  background-color: #f72210;
+}
+
+.subscribe-row {
+  position: relative;
+  padding: 2rem 0 4rem 0;
+  max-width: 95%;
+  margin: auto;
+}
+
+.s-img-one,
+.s-img-two {
+  position: absolute;
+  top: 0;
+}
+
+.s-img-two {
+  left: 25%;
+  top: 10%;
+}
+
+.trans-btn {
+  background: transparent;
+  color: #ddd;
+  display: inline;
+}
+
+.p-title {
+  font-weight: 300 !important;
+}
+
+@media (min-width: 1200px) {
+  .features-row img {
+    min-height: 300px;
+    object-fit: cover;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .navbar::before {
+    width: 66px;
+    position: absolute;
+    top: 0;
+    right: 0;
+    content: "";
+    display: block;
+    background-color: #f72210;
+    border-left: 12px solid white;
+    border-top: 13px solid #f72210;
+  }
+}
+
+.img-100 {
+  width: 100%;
+}
+
+@media (max-width: 800px) {
+  header .btn {
+    font-size: 0.9rem;
+    padding: 0;
+    position: relative;
+    top: 10px;
+  }
+
+  header .btn-col {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  header .fa {
+    padding-left: 0.7rem;
+    font-size: 1.2rem;
+  }
+
+  .top-stories-col {
+    order: 1;
+  }
+
+  .featured-stories-col {
+    order: 2;
+  }
+
+  .opinion-col {
+    order: 3;
+  }
+
+  .category-box {
+    margin-top: 1.4rem;
+  }
+
+  .newsletter-col {
+    margin-top: 1rem;
+  }
+
+  .newsletter-row {
+    padding-bottom: 1rem;
+  }
+
+  .subscribe-sec .story-title {
+    margin-top: 80%;
+  }
+
+  .copyright {
+    display: block;
```

Comment:
Consider using a `Bootstrap` class name for this selector element and in all other occurrences instead.

Bootstrap class names can be used instead. See [Display](https://getbootstrap.com/docs/4.1/utilities/display/) in bootstrap documentation.

> Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.

#### Please, lets do this for all other occurrences. :blush: